src/Compilers/VisualBasic/Portable/Symbols/Tuples/TupleErrorFieldSymbol.vb

```diff
@@ -94,7 +94,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
 
         Public Overloads Function Equals(other As TupleErrorFieldSymbol) As Boolean
             Return other Is Me OrElse
-                (other IsNot Nothing AndAlso Me._tupleFieldId = other._tupleFieldId AndAlso Me.ContainingType = other.ContainingType)
+                (other IsNot Nothing AndAlso
+                Me._tupleElementIndex = other._tupleElementIndex AndAlso
+                Me.ContainingType = other.ContainingType AndAlso
+                Me.Name = other.Name)
```

Comment:
I do not think that named, not named or partially named cases have a lot of differences. Perhaps i am missing something.
Will discuss.